examples/pr-in.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
Make it scsi3-reservations.py, similar to scsi2-reservations.py we already have. Also it would be better to merge all three examples in one file.